test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -1272,7 +1272,7 @@ INSTANTIATE_TEST_CASE_P(
     mkModelTc(pooling_avg_same_padding_multi_stride, "3:1:1:10", 10),
     mkModelTc(pooling_avg_valid_padding, "3:1:1:10", 10),
     mkModelTc(pooling_global_avg, "3:1:1:10", 10),
-    mkModelTc(pooling_global_max, "3:1:1:10", 10),
+    // mkModelTc(pooling_global_max, "3:1:1:10", 10),
```

Comment:
Wonder why this is commented

Reply:
my bad for the incomplete description. GLOBAL_MAX is still WIP.

Reply:
done.